File: tools/dbench/answers/python_haiku/s3/f1_users.py
l = open("users.txt").readlines()
u = []
for x in l:
    p = x.strip().split()
    if len(p) == 3:
        try:
            u.append(int(p[2]))
        except:
            pass
print(len(u))
print(sum(u) // len(u) if u else 0)

File: tools/dbench/answers/go_haiku/s2/f1_users.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	f, _ := os.Open("users.txt")
	defer f.Close()

	sc := bufio.NewScanner(f)
	cnt := 0
	sum := 0

	for sc.Scan() {
		parts := strings.Fields(sc.Text())
		if len(parts) != 3 {
			continue
		}
		age, err := strconv.Atoi(parts[2])
		if err != nil {
			continue
		}
		cnt++
		sum += age
	}

	fmt.Println(cnt)
	if cnt > 0 {
		fmt.Println(sum / cnt)
	}
}
